unblob/handlers/archive/zip.py

```diff
@@ -134,20 +147,12 @@ def calculate_chunk(
             end_of_zip = self._calculate_zipfile_end(file, start_offset)
         except MissingEOCDHeader:
             return
-
         file.seek(start_offset)
 
-        has_encrypted_files = False
-
         zip_content = file.read(end_of_zip - start_offset)
```

Comment:
Hmm, we shouldn't read the whole data into memory... I've opened #202 to not forget this issue